package.json

```diff
@@ -61,6 +61,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
+    "@folio/stripes-testing": "^4.5.0",

```

Comment:
This should go into devDeps since it's not required for actual module functionality